modules/node/src/redis/redis.provider.ts

```diff
@@ -21,6 +21,8 @@ export const redisClientFactory: FactoryProvider = {
   },
 };
 
+const RETRY_DELAY = 50;
```

Comment:
We can't set `RETRY_DELAY` too low. Setting this too low will result in the backend being bogged down in lock checks.